BUILD_INSTRUCTIONS.markdown

```diff
@@ -16,11 +16,17 @@ token after creating an account here:
 
 https://api.portal.trustedservices.intel.com/EPID-attestation
 
+Clone the Veracruz repository:
+
+```
+git clone git@github.com:veracruz-project/veracruz.git
```

Comment:
Doesn't this need to be a recursive clone?

Reply:
Yes, I think so.